src/conf.cpp

```diff
@@ -32,3 +37,12 @@ Config::Config() {
     this->outfile = NULL;
     this->device = NULL;
 }
+
+void Config::set_link_layer_type() {
```

Comment:
I'm just worried this relies a great deal upon the user to know what traffic they are dealing with.